azure-pipelines.yml

```diff
@@ -349,7 +348,6 @@ stages:
       enableSourceLinkValidation: false
       validateDependsOn:
       - Build_Windows_NT
```

Comment:
I don't think you want to / need to remove this here; otherwise it means that validate OSX starts before OSX finishes on public CI?

Reply:
There is just one Validate and there will be no OSX in internal build so not sure it should depend on a stage that won't happen?

![image](https://user-images.githubusercontent.com/7013027/88825451-e215d480-d1c7-11ea-8971-0491bd2a3e7c.png)